DevEdu.API/DevEdu.Business/Services/PaymentService.cs

```diff
@@ -23,8 +24,16 @@ public PaymentService(IPaymentRepository paymentRepository)
 
         public void UpdatePayment(int id, PaymentDto dto)
         {
+            var paymentInDb = _paymentRepository.GetPayment(id);
+            if(paymentInDb.IsDeleted == true)  
```

Comment:
1. possible NullReferenceException
2. what's the difference between paymentInDb.IsDeleted and paymentInDb.IsDeleted == true ?